pkg/restic/common.go

```diff
@@ -39,25 +39,16 @@ const (
 	InitContainer               = "restic-wait"
 	DefaultMaintenanceFrequency = 24 * time.Hour
 
-	podAnnotationPrefix       = "snapshot.velero.io/"
+	// Deprecated.
+	podAnnotationPrefix = "snapshot.velero.io/"
+
 	volumesToBackupAnnotation = "backup.velero.io/backup-volumes"
 )
 
-// PodHasSnapshotAnnotation returns true if the object has an annotation
-// indicating that there is a restic snapshot for a volume in this pod,
-// or false otherwise.
-func PodHasSnapshotAnnotation(obj metav1.Object) bool {
-	for key := range obj.GetAnnotations() {
-		if strings.HasPrefix(key, podAnnotationPrefix) {
-			return true
-		}
-	}
-
-	return false
-}
-
 // GetPodSnapshotAnnotations returns a map, of volume name -> snapshot id,
 // of all restic snapshots for this pod.
+// Deprecated: we will stop using pod annotation to determine restic
```

Comment:
This comment isn't accurate - should say that we're no longer using pod annotations to record restic snapshot IDs after they're taken.